lms/templates/main.html

```diff
@@ -85,30 +92,7 @@
 
   <%block name="headextra"/>
 
-<%
-  if theme_enabled() and not is_microsite():
-    header_extra_file = 'theme-head-extra.html'
```

Comment:
@mtyaka yes, thank you for pointing this out.  We're going to take a closer look at how old themes need to change in order to continue to work.  I think we are at the point where we will tell people how to convert their old themes into new themes, and old themes will not be supported any more.